packages/api/bin/serveUtils.js

```diff
@@ -3,70 +3,185 @@
 const indexer = require('@cumulus/es-client/indexer');
 const models = require('../models');
 const { getESClientAndIndex } = require('./local-test-defaults');
+const {
+  erasePostgresTables,
+} = require('./serve');
+const {
+  GranulesExecutionsPgModel,
+  RulePgModel,
+  GranulePgModel,
+  PdrPgModel,
+  ExecutionPgModel,
+  CollectionPgModel,
+  ProviderPgModel,
+  PostgresGranuleExecution,
+  translateApiCollectionToPostgresCollection,
+  translateApiProviderToPostgresProvider,
+  translateApiExecutionToPostgresExecution,
+  translateApiGranuleToPostgresGranule,
+  translateApiPdrToPostgresPdr,
+  translateApiRuleToPostgresRule,
+  getKnexClient,
+  localStackConnectionEnv,
+  envParams,
+  createTestDatabase
+} = require('@cumulus/db');
+const { log } = require('console');
+
+const migrationDir = 'node_modules/@cumulus/db/dist/migrations';
```

Comment:
Why is this not part of the require out of `@cumulus/db`?

Reply:
Great question. You are 100% correct.

Reply:
it's been updated, and pushed